.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
ESLint is a linting engine that helps make sure your code will compile and that you are following Standard JS coding practices.

It's optional - you don't have to install or use it if you don't want. I like it because it tells me if I've made a coding mistake that will prevent the app from compiling